server/src/main/java/io/crate/metadata/blob/BlobTableInfoFactory.java

```diff
@@ -51,14 +57,34 @@ public BlobTableInfoFactory(Settings settings, Environment environment) {
         this.globalBlobPath = BlobIndicesService.getGlobalBlobPath(settings);
     }
 
-    public BlobTableInfo create(RelationName ident, ClusterState clusterState) {
-        BlobTable blobTable = clusterState.metadata().getRelation(ident);
-        if (blobTable == null) {
-            throw new RelationUnknown(ident);
+    private IndexMetadata resolveIndexMetadata(String tableName, Metadata metadata) {
+        String indexName = BlobIndex.fullIndexName(tableName);
+        Index index;
+        try {
+            index = IndexNameExpressionResolver.concreteIndices(metadata, IndicesOptions.STRICT_EXPAND_OPEN, indexName)[0];
+        } catch (IndexNotFoundException ex) {
+            throw new RelationUnknown(indexName, ex);
         }
```

Comment:
This should probably follow through so we use the `schemas` based data if this doesn't work.

Reply:
thanks, updated here, restarted tests in https://github.com/crate/crate-qa/pull/336 and pointed rolling upgrade to this branch (https://github.com/crate/crate-qa/pull/337/commits/6885fdca963c6d34fc75659f39e07cb223137046)